onnxruntime/contrib_ops/rocm/bert/skip_layer_norm.cc

```diff
@@ -106,7 +109,8 @@ Status SkipLayerNorm<T>::ComputeInternal(OpKernelContext* ctx) const {
       (bias != nullptr) ? reinterpret_cast<const HipT*>(bias->Data<T>()) : nullptr,
       epsilon_,
       hidden_size,
-      static_cast<int>(element_count));
+      static_cast<int>(element_count),
+      tuning_);
```

Comment:
We should move the tuning logic outside of this function, that is, something like this:

```cpp
if (IsTuningEnabled) {
    static SkipLayerNormTunableOp<T> op;
    op.EnableTuning();
    SkipLayerNormParams<T> params(stream, output, input, skip, gamma, beta, bias, epsilon, ld, element_count);
    return op(&op_params);
}

return LaunchSkipLayerNormKernel(...);
```